File: game/desktop/src/main/java/com/quane/little/java/LittleDesktop.java
package com.quane.little.java;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.quane.little.game.Little;

public class LittleDesktop {

	public static void main(String[] args) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		config.title = "little";
		config.useGL20 = true;
		config.width = 1024;
		config.height = 768;
		new LwjglApplication(new Little(), config);
	}

}

File: game/desktop/src/main/scala/com/quane/little/game/LittleDesktop.scala
package com.quane.little.game

import com.badlogic.gdx.backends.lwjgl.{LwjglApplication, LwjglApplicationConfiguration}

/**
 *
 * @author Sean Connolly
 */
object LittleDesktop extends App {

  val Config = new LwjglApplicationConfiguration()
  Config.title = "little"
  Config.useGL20 = true
  Config.width = 1024
  Config.height = 768

  new LittleDesktop

}

class LittleDesktop {

  new LwjglApplication(new Little(), LittleDesktop.Config)

}